sdk/python/src/dagger/telemetry.py

```diff
@@ -146,8 +138,15 @@ def _prepare_env(self):
         # The default is a NoOpProvider.
         os.environ.setdefault(OTEL_PYTHON_TRACER_PROVIDER, "sdk_tracer_provider")
 
-        # TODO: The INSECURE env vars should be set by the shim next to ENDPOINT
-        # rather than in the SDK.
+        # TODO: The following env vars should be set by the shim rather than in the SDK.
+
+        for exporter in (
+            OTEL_TRACES_EXPORTER,
+            OTEL_LOGS_EXPORTER,
+            OTEL_METRICS_EXPORTER,
+        ):
+            os.environ.setdefault(exporter, "otlp")
```

Comment:
These env vars used to be added by the engine. Took me a beat to debug and understand what was wrong.